bees/discordbee/discordbee.go

```diff
@@ -141,10 +141,10 @@ func (mod *DiscordBee) Action(action bees.Action) []bees.Placeholder {
 		if err != nil {
 			mod.LogErrorf("Unable to send message: %v", err)
 		}
-        message, err2 := mod.discord.ChannelMessageCrosspost(channelID, message.ID)
-        if err2 != nil {
-                mod.LogErrorf("Unable to publish message: %v", err2)
-        }
+		message, err2 := mod.discord.ChannelMessageCrosspost(channelID, message.ID)
+		if err2 != nil {
+				mod.LogErrorf("Unable to publish message: %v", err2)
```

Comment:
The indentation is still broken here. Did you use `gofmt` or did you do this manually?

Reply:
Just a heads up: I'll fix this for you and push to your branch.

Reply:
Oh? Sorry, seems good for me :-/  

I did it manually after discovering this was because of my IDE, i manually copy/paste "tabs" char in an old vi editor to avoid any visual glitch. With no luck it seems, I really have something in the eyes this week end 😅